README.md

```diff
@@ -15,6 +15,14 @@ documentation.  Confusingly there are several open source projects called
 the one you want.  We aren't certain exactly what version is needed - it
 seems Sphinx 0.6.4 is too old but 1.1.3 is new enough.
 
+For building a pdf version of documentation,you will need to install rst2pdf
+
+https://github.com/rst2pdf/rst2pdf
```

Comment:
We should recommend installing from pip; installing python packages from a git master is frowned upon for various reasons. Also worth pointing out to people that `rsf2pdf` doesn't support python3 at the moment.